### PR TITLE
Support data asset type in wiki

### PR DESCRIPTION
**Pull request by lykelly19**

### CODE REVIEW

**Review by elesel — DISMISSED** [review]

**Review by vizhur — DISMISSED** [review]

File: scripts/docs/generate_asset_documentation.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 SUPPORTED_ASSET_TYPES = [AssetType.ENVIRONMENT, AssetType.COMPONENT, AssetType.MODEL, AssetType.DATA]
 DEFAULT_CATEGORY = "Uncategorized"
+ALREADY_PLURALIZED_TYPES = ["data"]
```
Comment (elesel) [inline scripts/docs/generate_asset_documentation.py:17]: Consider making a dict of type to plural values so they can all be handled consistently.

File: scripts/docs/generate_asset_documentation.py [hunk 2]
```diff
@@ -68,7 +74,7 @@ def filename(self) -> str:
     @property
     def directory(self) -> Path:
         """Return asset directory."""
-        return Path(f"{self.type}s")
+        return Path(f"{self.type}")
```
Comment (vizhur) [inline scripts/docs/generate_asset_documentation.py:77]: pluralized_type

**Comment by vizhur** [comment]
https://github.com/Azure/azureml-assets/wiki/component**s**-documentation is now generated as component-documentation.

File: scripts/docs/generate_asset_documentation.py [hunk 3]
```diff
@@ -378,21 +412,22 @@ def __init__(self, name: str, type: str, parent=None):
...
+        return (parent_category + "-" + self._name) if parent_category else (self._type)
 
     @property
     def _doc_name(self):
         return f"{self._category_full_path}-documentation".replace(" ", "-")
 
     @property
     def _doc_full_path_name(self):
-        return f"{self._type}s/{self._doc_name}.md"
+        return f"{self._type}/{self._doc_name}.md"
```
Comment (vizhur) [inline scripts/docs/generate_asset_documentation.py:430]: _pluralized_type

**Pull request merged**